meson_options.txt

```diff
@@ -4,3 +4,8 @@ option(
   type: 'string',
   value: '/usr/share/profile.d'
 )
+option(
+  'tmpfilesdir',
+  description: 'Directory for tmpfilesdir files to be installed by root user',
```

Comment:
I am curious about the *installed by root user* part. I thought that this option is meant to be useful when installing to a non-system prefix, and to avoid requiring `root` access while doing that.

Reply:
My impression was that there is no current way to configure user specific tmpfiles.d.  They will always need root to be effective.

If that's not the case, this can obviously be updated.  I didn't look into it a ton, so it may totally be possible.

Reply:
Oh, never mind, it's right there in the synopsis.  I wonder if I was blind to it, or if it's since been better documented:
```
       /etc/tmpfiles.d/*.conf
       /run/tmpfiles.d/*.conf
       /usr/lib/tmpfiles.d/*.conf

       ~/.config/user-tmpfiles.d/*.conf
       $XDG_RUNTIME_DIR/user-tmpfiles.d/*.conf
       ~/.local/share/user-tmpfiles.d/*.conf
       ...
       /usr/share/user-tmpfiles.d/*.conf
```